compose/types/rest.go

```diff
@@ -1,8 +1,14 @@
 package types
 
+import (
```

Comment:
Can we move this somewhere else? Why "rest"? This part of the app should not be aware of that layer at all.